assets/js/components/consent-mode/WPConsentAPIRequirements.js

```diff
@@ -59,6 +60,15 @@ export default function WPConsentAPIRequirements() {
 	const { installActivateWPConsentAPI, activateConsentAPI } =
 		useDispatch( CORE_SITE );
 
+	// Check for API errors in the user store, mostly to cover
+	// the case if user is offline, but also any other potential
+	// case that API fails to execute. As `installActivateWPConsentAPI`
+	// action will invoke fetch for nonce, and when offline this will fail
+	// and API error will be fed to the `CORE_USER` store.
+	const installWPConsentAPIError = useSelect( ( select ) =>
+		select( CORE_USER ).getErrors()
```

Comment:
Unfortunately, this will not work correctly because this will return all errors for all actions and selectors that has been thrown on the dashboard. To get an error specifically for this matter, we need to call `getErrorForAction( ... )` instead.

Reply:
Thanks! that's good to know.
Updated it to use `getErrorForAction`